### PR TITLE
Update Emoji-cheat-sheet link

### LINKED ISSUE #1992: Not matching URLs for emoji-cheat-sheet
### What article on docs.github.com is affected?
https://docs.github.com/en/free-pro-team@latest/github/writing-on-github/basic-writing-and-formatting-syntax#using-emoji

### What part(s) of the article would you like to see updated?
The link in the section points to https://emoji-cheat-sheet.com (Which actually is https://www.webfx.com/tools/emoji-cheat-sheet/) while the Guide you get linked to when clicking "Styling with Markdown is supported" in the editor does link to a [GitHub Repo and its readme](https://github.com/ikatyang/emoji-cheat-sheet/blob/master/README.md)

The docs should imo represent the guide in terms of links which means it should link to the repository.
This also may have the advantage that only actually supported emojis will be shown.

### PR DESCRIPTION
**Pull request by Andre601**
### Why:

closes #1992 

While I did explain in the above issue why, will I still say the reason here for the sake or documentation.
The doc page explaining emojis links to https://emoji-cheat-sheet.com, while the Guide about GFM and emojis is linking to a repository which seems to be updated.
Since people may get to the docs through the guide would those two different URLs only cause confusion as to which one to look at for up-to-date info.

I now changed the link to point to the aforementioned repository as there is no guarantee (unless GitHub would do this) that the Guide would be updated AND that the emoji-cheat-sheet page only shows emojis supported on GitHub and not also extra ones not available here.

### What's being changed:

Section "Using Emojis" of `content/github/writing-on-github/basic-writing-and-formatting-syntax.md`

Diff:
```diff
- For a full list of available emoji and codes, check out [emoji-cheat-sheet.com](http://emoji-cheat-sheet.com).
+ For a full list of available emoji and codes, check out [the Emoji-Cheat-Sheet](https://github.com/ikatyang/emoji-cheat-sheet/blob/master/README.md).
```

### Check off the following:
- [x] All of the tests are passing.
- [x] I have reviewed my changes in staging. (look for the **deploy-to-heroku** link in your pull request, then click **View deployment**)
- [x] For content changes, I have reviewed the [localization checklist](https://github.com/github/docs/blob/main/contributing/localization-checklist.md)
- [x] For content changes, I have reviewed the [Content style guide for GitHub Docs](https://github.com/github/docs/blob/main/contributing/content-style-guide.md).

### CODE REVIEW

**Comment by janiceilene** [comment]
@Andre601 Thanks so much for opening a PR! I'll get this triaged for review ⚡

**Review by BxndoBen — APPROVED** [review]

**Review by BxndoBen — APPROVED** [review]

File: content/github/writing-on-github/basic-writing-and-formatting-syntax.md [hunk 1]
```diff
@@ -215,7 +215,7 @@ You can add emoji to your writing by typing `:EMOJICODE:`.
 
 Typing `:` will bring up a list of suggested emoji. The list will filter as you type, so once you find the emoji you're looking for, press **Tab** or **Enter** to complete the highlighted result.
 
-For a full list of available emoji and codes, check out [emoji-cheat-sheet.com](http://emoji-cheat-sheet.com).
```
Comment (BxndoBen) [inline content/github/writing-on-github/basic-writing-and-formatting-syntax.md:218]: emojicode

**Review by BxndoBen — APPROVED** [review]

File: content/github/writing-on-github/basic-writing-and-formatting-syntax.md (hunk 1 above)
Comment (Andre601) [reply content/github/writing-on-github/basic-writing-and-formatting-syntax.md:218]: I would appreciate if you would not constantly make approves and also only submit suggestions/change requests that do make sense here.

**Comment by Andre601** [comment]
Is this still being checked?

**Comment by janiceilene** [comment]
It is! Thanks for your patience! Our small team is working our way through reviewing all of the amazing contributions ✨

**Comment by Andre601** [comment]
Yes GitHub Actions, this PR is still active.
Thanks for checking :)

**Review by BxndoBen — APPROVED** [review]
content/github/writing-on-github/basic-writing-and-formatting-syntax.md

**Comment by Andre601** [comment]
> content/github/writing-on-github/basic-writing-and-formatting-syntax.md

I mentioned this before and I mention this again: I and probably anyone else following this PR would appreciate it, if you wouldn't make PR approvals over and over again. While this was just one approval this time was it still not nesessary to do as there were no changes whatsoever that justified it.

**Review by janiceilene — APPROVED** [review]
Thanks for making this change @Andre601 and I'm so sorry it got hit with spammers 💛 I'll update the branch and get this merged once all tests are passing!

**Comment by Andre601** [comment]
> Thanks for making this change @Andre601 and I'm so sorry it got hit with spammers 💛 I'll update the branch and get this merged once all tests are passing!

That's okay as long as those people are dealt with. I reported one user already and also gave the support-team the hint, that they should also check out the other users I mentioned, so hopefully those people will be gone soon enough.

I honestly wish I would understand even half of all the checks/actions made on this PR 😅

**Pull request merged**